src/actions.js

```diff
@@ -166,6 +166,8 @@ export function receiveInvoices(invoicesData) {
   }
 }
 
+// Invoice download action
```

Comment:
This comment is inappropriate here. Moved the corresponding action def out of here but the comment had stayed on. Will change